plugins/scaffolder-backend-module-gitlab/report.api.md

```diff
@@ -88,6 +88,20 @@ export const createGitlabProjectDeployTokenAction: (options: {
   }
 >;
 
+// @public
+export const createGitlabProjectMigrateAction: (options: {
```

Comment:
I'm wondering if we should just stop exporting these actions individually now, as theres no real need with the new backend system, that you just add the module and it registers them for you. It's just going to be removed pretty soon with the removal of all the old backend system work.